utils/schemaValidator.js

```diff
@@ -0,0 +1,19 @@
+module.exports.valid = function (req, res, next, schema) {
+  req.checkBody(schema)
+  const errors = req.validationErrors()
+  if (errors) {
+    res.status(500).send(errors)
+    return
+  }
```

Comment:
you can also simplify this "if statement"

Reply:
simplified, thanks!